src/application/wizard/service.ts

```diff
@@ -201,7 +201,10 @@ export class WizardService {
         const supportingFiles = (await this.fileService.getSupportingFiles(submissionId)).filter(
             file => !file.isCancelled() && !file.isDeleted(),
         );
-
-        return { ...submission, manuscriptFile, supportingFiles } as Submission;
+        const files: FileDetails = {
+            manuscriptFile,
+            supportingFiles,
+        };
+        return { ...submission, files } as Submission;
```

Comment:
Not related to this PR, and I only just noticed this, but we shouldn't be doing. We should be using the constructor

Reply:
Just tested this. This should raise a compiler error due to required properties, but it doesn't: 
`return { id: submission.id } as Submission;`